src/charm.py

```diff
@@ -213,6 +244,8 @@ def _check_ready(self) -> bool:
             "ingress": self._get_public_uri(),
             "penpot container": self.container.can_connect(),
         }
+        if self._get_penpot_oauth_config():
+            requirements["smtp"] = self._get_smtp_credentials()
```

Comment:
Why do we set smtp as required when oauth oauth is configured ? Might require a small comment

Reply:
Yes, SMTP is required for the OpenID Connect-based registration process. Added a comment, thanks!